packages/zarf-registry/zarf.yaml

```diff
@@ -106,6 +106,18 @@ variables:
     description: Decides if host port or host network should be used with the registry proxy
     default: "false"
 
+  - name: REGISTRY_SECURITY_CONTEXT_RUN_AS_USER
```

Comment:
nit: I believe we should validate inputs with a `pattern` on these items. (positive integer perhaps?)